src/zombies.test.ts

```diff
@@ -1,30 +1,63 @@
 import { ok } from "node:assert/strict";
 import { test } from "node:test";
 
-const createRoom = (capacity: number) => {
-  const _capacity = capacity;
+const createRoom = (initCapacity: number) => {
+  let _capacity = initCapacity;
 
   return {
-    isFull: () => true,
+    isFull: () => (_capacity === 0 ? true : false),
+    notFull: () => (_capacity === 1 ? true : false),
+    isEmpty: () => _capacity === 0,
+    addZombie: () => (_capacity ? (_capacity--, true) : null),
+    removeZombie: () => (_capacity ? (_capacity++, true) : null),
   };
 };
 
 test("room is not full", () => {
-  const room = createRoom(0);
+  const roomCapacity = createRoom(1);
+  const result = roomCapacity.isFull();
 
-  const result = room.isFull();
+  ok(!result);
+});
+
+test("empty room that fits one zombie is not full", () => {
+  const roomCapacity = createRoom(1);
+  const result = roomCapacity.notFull();
 
   ok(result);
 });
 
-test.skip("empty room that fits one zombie is not full", () => {});
+test("empty room cannot fit any zombies", () => {
+  const roomCapacity = createRoom(0);
+  const result = roomCapacity.isEmpty();
 
-test.skip("empty room cannot fit any zombies", () => {});
+  ok(result);
+});
 
-test.skip("one-roomer becomes full when a zombie is added", () => {});
+test("one-roomer becomes full when a zombie is added", () => {
+  const roomCapacity = createRoom(1);
+  // when adding a zombie, the capacity of the room will decrease by 1.
+  roomCapacity.addZombie();
+  const result = roomCapacity.isFull();
+
+  ok(result);
+});
 
-test.skip("two-roomer is not full when a zombie is added", () => {});
+test("two-roomer is not full when a zombie is added", () => {
+  const roomCapacity = createRoom(2);
+  roomCapacity.addZombie();
+  const result = roomCapacity.notFull();
```

Comment:
There's no need for a `notFull()` method as it's just the negative of `isFull()`.